modules/nf-core/deepvariant/tests/main.nf.test

```diff
@@ -0,0 +1,108 @@
+nextflow_process {
+
+    name "Test Process DEEPVARIANT"
+    script "../main.nf"
+    process "DEEPVARIANT"
+
+    tag "modules"
+    tag "modules_nfcore"
+
+    test("Test deepvariant") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                      file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam'], checkIfExists: true),
+                      file(params.test_data['homo_sapiens']['illumina']['test_paired_end_sorted_bam_bai'], checkIfExists: true),
```

Comment:
Rewrite all the input files with `params.modules_testdata_base_path`
```suggestion
                      file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true),
                      file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam.bai', checkIfExists: true),
```

Reply:
@Joon-Klaps where can I find the files available for testing so I construct the paths, please?

Reply:
Found it ;) https://github.com/nf-core/test-datasets/tree/modules/data/genomics/homo_sapiens

Reply:
All files are in this location https://github.com/nf-core/modules/blob/master/tests/config/test_data.config . 
Note that you'll have to change `params.test_data_base` to `params.modules_testdata_base_path`.